src/Form/LegacyConsumer/templates/base.html.php

```diff
@@ -1,9 +1,9 @@
-<?php /** @var Give\Framework\FieldsAPI\Field $field */ ?>
+<?php /** @var Give\Framework\FieldsAPI\Field|\Give\Framework\FieldsAPI\Text $field */ ?>
 <?php /** @var string $typeAttribute */ ?>
 <input
 	type="<?php echo $typeAttribute; ?>"
 	name="<?php echo $field->getName(); ?>"
-	placeholder="<?php echo $field->getLabel(); ?>"
+	placeholder="<?php echo $field->getPlaceHolder(); ?>"
```

Comment:
Typo here:

```suggestion
	placeholder="<?php echo $field->getPlaceholder(); ?>"
```

Reply:
Also, @JasonTheAdams did we have the label set here as the placeholder for the sake of the multi-step template?

Reply:
@kbjohnson90 Any thought on @knowler comment 👆